amo2kinto/importer.py

```diff
@@ -117,6 +117,15 @@ def main(args=None):
 
     kinto_client = cli_utils.create_client_from_args(args)
 
+    # Check if the schema capability is activated
+    body, headers = kinto_client.session.request('get', '/')
+    if 'schema' not in body.get('capabilities', {}):
+        logger.warn('\t --- Importing without schema validation --- \n')
+        logger.warn('The kinto server at {}/ does not supports schema '
+                    'validation.\nPlease set the following config: ``'
+                    'kinto.experimental_collection_schema_validation = true``'
```

Comment:
I don't think this is going to be obvious for the one executing the script. We could just say `The server won't validate the records against the collection JSON schema. More information http://kinto.readthedocs.io/en/stable/api/1.x/collections.html?highlight=json%20validation#collection-json-schema`